test/suites/dev/test-staking/test-staking-consts.ts

```diff
@@ -70,6 +70,24 @@ describeSuite({
       let nonce = await context
         .viem()
         .getTransactionCount({ address: randomAccount.address as `0x${string}` });
+
+      // Delegate to alice to avoid the 'no nimbus keys available to manual seal' error.
+      // It seems that if we don't do this, somehow alice is not recognized as the author
+      // and is not able to produce the next block.
```

Comment:
It's a new bug then, we should understand what happens

Reply:
This was being an issue when dealing with the relay block number, now I removed the extra delegation and the test works fine as it used to do before this PR.